Sources/Plasma/Apps/plClient/win32/plClient_Win.cpp

```diff
@@ -99,14 +99,12 @@ void plClient::IChangeResolution(int width, int height)
     HMONITOR monitor = MonitorFromWindow(fWindowHndl, MONITOR_DEFAULTTONULL);
     if (!monitor)
         return;
-    MONITORINFOEXW moninfo;
-    memset(&moninfo, 0, sizeof(moninfo));
+    MONITORINFOEXW moninfo = {};
```

Comment:
```suggestion
    MONITORINFOEXW moninfo{};
```

Reply:
i was thinking of adding the = but wasnt sure if it would change the code